channel/consensus_channel/leader_channel_test.go

```diff
@@ -197,3 +197,28 @@ func TestRestrictedFollowerMethods(t *testing.T) {
 		t.Errorf("Expected error when calling Receive as a leader, but found none")
 	}
 }
+
+func TestLeaderIncorrectlyAddressedProposals(t *testing.T) {
```

Comment:
On `main`, the leader_channel  tests have been updated. As follow-up work, this test could be written as
```
	{
		msg := "err:wrong channel"
		p := populatedQueue()[2]
		p4 := createAdd(p.TurnNum+10, types.Destination{11})
		p4.ChannelID = "wrong channel"
		counterP := bobSignedProposal(p.Vars, p4)
		t.Run(msg, testUpdateConsensusErr(counterP, ErrIncorrectChannelID))
	}
```

Reply:
Noted!